app/src/main/res/values/strings.xml

```diff
@@ -302,4 +302,27 @@
     <string name="dismiss_temporary">Not now</string>
     <string name="update_available">New update available</string>
     <string name="update_available_description">A new version (%s) is available for download.</string>
+
+    <string name="installer_result_failed">Failed to install</string>
+    <string name="installer_result_cancelled">Cancelled</string>
+    <string name="installer_result_blocked">Blocked</string>
+    <string name="installer_result_conflict">Conflict</string>
+    <string name="installer_result_incompatible">Incompatible</string>
+    <string name="installer_result_invalid">Invalid</string>
+    <string name="installer_result_storage_issue">Storage issue</string>
+    <string name="installer_result_timeout">Timeout</string>
+    <string name="installer_result_success">Success</string>
+    <string name="installer_result_unknown">Unknown</string>
+
+    <string name="installer_result_failed_description">The installation failed for an unknown reason.</string>
+    <string name="installer_result_aborted_description">The installation was manually cancelled.</string>
+    <string name="installer_result_blocked_description">The installation was blocked.</string>
+    <string name="installer_result_conflict_description">An existing installation of the app prevents the installation. Reinstall by uninstalling the app and try again?</string>
+    <string name="installer_result_incompatible_description">The app is incompatible with this device.</string>
+    <string name="installer_result_invalid_description">The app is invalid. Reinstall by uninstalling the app and try again?</string>
```

Comment:
```suggestion
    <string name="installer_result_invalid_description">The app is invalid. Uninstall the current installation and try again?</string>
```

Reply:
@Axelen123 A problem with this one is that an app we try to install can be invalid, but no installation of the app already exists. Do you have any suggestions?

Reply:
Is there anything stopping you from checking if the app is installed and showing a different message if it is?

Reply:
I am not sure how I would restructure the code. Any suggestions? Alternative I could say:

```diff
+ Uninstall any existing installation
- Uninstall the current installation
```

Reply:
Maybe a function that returns a string resource could work: https://kotlinlang.org/docs/enum-classes.html#anonymous-classes

We could call it with a class containing the information we need, but it might be better to just change the wording instead.